hugo/layouts/engine/single.html

```diff
@@ -71,6 +71,146 @@ <h4 class="title-line line-green">Familiar APIs</h4>
                     </div>
                 </div>
             </div>
+        <div class="row">
```

Comment:
These tabs are not working in my Chrome. Are their purpose to jump to a certain tab? If so, Slick offers a way to `go to certain tab`; if it should be a filter, Slick also offer that feature, but it could be less trivial.
Also: does the _autoplay_ play well with _jump-to_ feature? I think it can be a UX problem.